### PR TITLE
[easy] Rename the package name of sqlalchemy-enum-tables

### PR DESCRIPTION
**Pull request by ttung**
### Description
Some tools choke on package names that have spaces.  Rename this accordingly.

### Test plan
None.

### CODE REVIEW

File: third-party/sqlalchemy-enum-tables/setup.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def reqs():
 
 
 st.setup(
-	name = "SqlAlchemy Enum Tables",
+	name = "SqlAlchemy-Enum-Tables",
```
Comment (phoenixAja) [inline third-party/sqlalchemy-enum-tables/setup.py]: is this normal python package name formatting? I thought it would be like `sqlalchemy_enum_tables`

**Review by phoenixAja — APPROVED** [review]

File: third-party/sqlalchemy-enum-tables/setup.py (hunk 1 above)
Comment (ttung) [reply third-party/sqlalchemy-enum-tables/setup.py]: I can do that.

Comment (phoenixAja) [reply third-party/sqlalchemy-enum-tables/setup.py]: haha ok, but not an emergency

Comment (ttung) [reply third-party/sqlalchemy-enum-tables/setup.py]: Fixed!

Comment (phoenixAja) [reply third-party/sqlalchemy-enum-tables/setup.py]: if i could approve again i would haha

**Pull request merged**